msteams-platform/graph-api/rsc/resource-specific-consent.md

```diff
@@ -144,6 +200,17 @@ Once the app has been installed to a team, you can use [Graph Explorer](https://
 > - Make a **GET** call to the following endpoint: `https://graph.microsoft.com/beta/groups/{teamGroupId}/permissionGrants`. The clientAppId field in the response will map to the appId specified in the Teams app manifest.
   ![Graph explorer response to GET call.](../../assets/images/graph-permissions.png)
 
+### Check your app for added RSC permissions in a chat
+
+> [!div class="checklist"]
+>
+>- Get the chat thread id from the Teams web client.
+> - In the Teams web client, select **Chat** from the far left nav bar.
+> - Select the chat where the app is installed from the drop-down menu.
+> - Copy the web URL and save the chat thread id from the string.
+> - Log into **Graph Explorer**.
+> - Make a **GET** call to the following endpoint: `https://graph.microsoft.com/beta/chats/{chatId}/permissionGrants`. The clientAppId field in the response will map to the appId specified in the Teams app manifest.
+
```

Comment:
Can we also add a quick see also over here - point to listing details of apps installed in a chat.

Reply:
I changed the one for team as well. Is that correct?